Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2211,6 +2282,25 @@ void CGContextClearRect(CGContextRef context, CGRect rect) {
         this->PopEndDraw();
     });
 
+    // If the context has requested antialiasing other than the defaults we now need to update the device context.
+    if (CurrentGState().shouldAntialias != _kCGTrinaryDefault || allowsAntialiasing != _kCGTrinaryDefault) {
```

Comment:
It would probably be cleaner to just use a bool for these values and some sort of flag that gets set whenever one of these values is changed